fw/http_parser.c

```diff
@@ -7276,8 +7273,9 @@ do {									\
 #undef __NEXT_TEMPL_STATE
 
 	__FSM_STATE(I_GMT) {
-		H2_TRY_STR_BY_REF("gmt",
-			&&I_GMT, st[parser->date.type][parser->date.pos + 1]);
+		H2_TRY_STR_LAMBDA("gmt", {
+			__FSM_I_JMP(I_Res);
```

Comment:
It is known that I_GMT is followed by I_Res.

Reply:
It seems you're right, but could you please then fix the same place for the HTTP/1 parser and write a 2 `gmt` test cases for both the parsers?